Globant Abstract factory/src/com/globant/NoSQLFactory.java

```diff
@@ -0,0 +1,14 @@
+package com.globant;
+
+public class NoSQLFactory extends AbstractFactory {
+
+    @Override
+    public Connection getConnection(String connectionType) {
+        if (connectionType.equalsIgnoreCase("MYSQL")) {
```

Comment:
Change the condition order, it should be
```suggestion
        if ("MYSQL".equalsIgnoreCase(connectionType)) {
```
Also, if you are getting MYSQL, why are you returning a NoSql connection?